### PR TITLE
Endpoint initiated uncommon remote scheduled task creation

### PR DESCRIPTION
**Pull request by OmriItzhak**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-12080)
## Description
New playbook and trigger to handle the alert "Uncommon remote scheduled task creation".

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml [hunk 1]
```diff
@@ -0,0 +1,1344 @@
...
+    isautoswitchedtoquietmode: false
+  "67":
+    id: "67"
+    taskid: e2846c17-8044-43e0-881e-17219cfa784c
+    type: regular
+    task:
+      id: e2846c17-8044-43e0-881e-17219cfa784c
+      version: -1
+      name: Notify to War Room - Scheduled Task Disabled
+      description: Prints text to war room (Markdown supported)
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml:812]:
```suggestion
      description: Prints text to War Room (Markdown supported).
```

File: Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml [hunk 2]
```diff
@@ -0,0 +1,1344 @@
...
+    isautoswitchedtoquietmode: false
+  "74":
+    id: "74"
+    taskid: 4be27c4b-ad83-46b4-868b-795a35647cd0
+    type: regular
+    task:
+      id: 4be27c4b-ad83-46b4-868b-795a35647cd0
+      version: -1
+      name: Get remote endpoint details
+      description: Gets a list of endpoints, according to the passed filters. If there are no filters, all endpoints are returned. Filtering by multiple fields will be concatenated using AND condition (OR is not supported). Maximum result set size is 100. Offset is the zero-based number of endpoint from the start of the result set (start by counting from 0).
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml:1018]:
```suggestion
      description: Gets a list of endpoints, according to the passed filters. If there are no filters, all endpoints are returned. Filtering by multiple fields will be concatenated using the AND condition (OR is not supported). Maximum result set size is 100. Offset is the zero-based number of the endpoint from the start of the result set (start by counting from 0).
```

File: Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml [hunk 3]
```diff
@@ -0,0 +1,1344 @@
...
+    isautoswitchedtoquietmode: false
+  "75":
+    id: "75"
+    taskid: a2bad801-d912-45ec-8ac6-5ece49400caa
+    type: condition
+    task:
+      id: a2bad801-d912-45ec-8ac6-5ece49400caa
+      version: -1
+      name: Is the XSIAM agent installed on the remote endpoint?
+      description: Checks whether the XSIAM agent is installed on the remote endpoint.
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml:1053]:
```suggestion
      description: Checks whether the Cortex XSIAM agent is installed on the remote endpoint.
```

File: Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml [hunk 4]
```diff
@@ -0,0 +1,1344 @@
...
+    isautoswitchedtoquietmode: false
+  "76":
+    id: "76"
+    taskid: 36164015-8f07-4e5b-873d-024f66adb228
+    type: condition
+    task:
+      id: 36164015-8f07-4e5b-873d-024f66adb228
+      version: -1
+      name: Is the XSIAM agent install and connected on the remote endpoint?
+      description: Checks whether the XSIAM agent is installed and connected on the remote endpoint.
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml:1094]:
```suggestion
      description: Checks whether the Cortex XSIAM agent is installed and connected on the remote endpoint.
```

File: Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task_README.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+Analysis:
+
+- The playbook verifies whether the causality process is signed. If the process is not signed, it proceeds with remediation actions; otherwise, it continues investigating the alert.
+
+Investigation:
+
+During the alert investigation, the playbook will perform the following:
+
+- Searches for related XSIAM alerts on the endpoint that use the following MITRE techniques to identify malicious activity: T1202 - Indirect Command Execution, T1021 - Remote Services.
+- Searches for related XSIAM agent alerts on the remote endpoint, to determine if the creation of the scheduled task is part of an attack pattern.
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task_README.md:14]:
```suggestion
- Searches for related Cortex XSIAM alerts on the endpoint that use the following MITRE techniques to identify malicious activity: T1202 - Indirect Command Execution, T1021 - Remote Services.
- Searches for related Cortex XSIAM agent alerts on the remote endpoint, to determine if the creation of the scheduled task is part of an attack pattern.
```

File: Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task_README.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+- Searches for suspicious command-line parameters indicating a malicious scheduled task.
+
+Remediation:
+
+- Automatically disable the malicious scheduled task on the remote host.
+- Terminate the causality process.
+- Automatically Close the alert.
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task_README.md:21]:
```suggestion
- Automatically close the alert.
```

File: Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task_README.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+
+* Command-Line Analysis
+
+### Integrations
+
+* CortexCoreIR
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task_README.md:33]:
```suggestion
Command-Line Analysis

### Integrations

CortexCoreIR
```

File: Packs/Core/ReleaseNotes/3_0_96.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+
+- Searches for related XSIAM alerts on the endpoint that use the following MITRE techniques to identify malicious activity: T1202 - Indirect Command Execution, T1021 - Remote Services.
+- Searches for related XSIAM agent alerts on the remote endpoint, to determine if the creation of the scheduled task is part of an attack pattern.
+- Searches for suspicious command-line parameters indicating a malicious scheduled task.
+
+Remediation:
+
+- Automatically disable the malicious scheduled task on the remote host.
+- Terminate the causality process.
+- Automatically Close the alert.
```
Comment (ShirleyDenkberg) [inline Packs/Core/ReleaseNotes/3_0_96.md:26]: Is it necessary to list all this information in the Release Notes?  It is in the yml file and the Readme file.

File: Packs/Core/ReleaseNotes/3_0_96.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+Analysis:
+
+- The playbook verifies whether the causality process is signed. If the process is not signed, it proceeds with remediation actions; otherwise, it continues investigating the alert.
+
+Investigation:
+
+During the alert investigation, the playbook will perform the following:
+
+- Searches for related XSIAM alerts on the endpoint that use the following MITRE techniques to identify malicious activity: T1202 - Indirect Command Execution, T1021 - Remote Services.
+- Searches for related XSIAM agent alerts on the remote endpoint, to determine if the creation of the scheduled task is part of an attack pattern.
```
Comment (ShirleyDenkberg) [inline Packs/Core/ReleaseNotes/3_0_96.md:19]:
```suggestion
- Searches for related Cortex XSIAM alerts on the endpoint that use the following MITRE techniques to identify malicious activity: T1202 - Indirect Command Execution, T1021 - Remote Services.
- Searches for related Cortex XSIAM agent alerts on the remote endpoint, to determine if the creation of the scheduled task is part of an attack pattern.
```

File: Packs/Core/ReleaseNotes/3_0_96.md (hunk 8 above)
Comment (ShirleyDenkberg) [inline Packs/Core/ReleaseNotes/3_0_96.md:26]:
```suggestion
- Automatically close the alert.
```

File: Packs/Core/ReleaseNotes/3_0_96.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+- Automatically disable the malicious scheduled task on the remote host.
+- Terminate the causality process.
+- Automatically Close the alert.
+
+
+#### Triggers Recommendations
+
+##### New: A source host created uncommon remote scheduled task
+
+- New: This trigger is responsible for handling 'Uncommon remote scheduled task creation' alerts.
```
Comment (ShirleyDenkberg) [inline Packs/Core/ReleaseNotes/3_0_96.md:33]:
```suggestion
New: This trigger is responsible for handling 'Uncommon remote scheduled task creation' alerts.
```

File: Packs/Core/ReleaseNotes/3_0_96.md [hunk 11]
```diff
@@ -0,0 +1,33 @@
+
+#### Playbooks
+
+##### New: A source host created uncommon remote scheduled task
+
+- New: This playbook handles "Uncommon remote scheduled task creation" alerts, which are generated on the source host that created the remote scheduled task.
```
Comment (ShirleyDenkberg) [inline Packs/Core/ReleaseNotes/3_0_96.md:6]:
```suggestion
New: This playbook handles "Uncommon remote scheduled task creation" alerts, which are generated on the source host that created the remote scheduled task.
```

**Comment by ShirleyDenkberg** [comment]
@altmannyarden Doc review completed.

File: Packs/Core/ReleaseNotes/3_0_96.md (hunk 8 above)
Comment (OmriItzhak) [reply Packs/Core/ReleaseNotes/3_0_96.md:26]: yes

File: Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task_README.md [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+During the alert investigation, the playbook will perform the following:
+
+- Searches for related Cortex XSIAM alerts on the endpoint that use the following MITRE techniques to identify malicious activity: T1202 - Indirect Command Execution, T1021 - Remote Services.
+- Searches for related Cortex XSIAM agent alerts on the remote endpoint, to determine if the creation of the scheduled task is part of an attack pattern.
+- Searches for suspicious command-line parameters indicating a malicious scheduled task.
+
+Remediation:
+
+- Automatically disable the malicious scheduled task on the remote host.
+- Terminate the causality process.
```
Comment (altmannyarden) [inline Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task_README.md:20]: Is the termination not automatically?

File: Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task_README.md [hunk 13]
```diff
@@ -0,0 +1,65 @@
+This playbook handles "Uncommon remote scheduled task creation" alerts, which are generated on the source host that created the remote scheduled task.
```
Comment (altmannyarden) [inline Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task_README.md:1]: need to fix first sentence as it handles single alert alert not alerts and 
is not are

File: Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml [hunk 14]
```diff
@@ -0,0 +1,1344 @@
...
+    isautoswitchedtoquietmode: false
+  "29":
+    id: "29"
+    taskid: b6d11f6e-a28a-459a-8004-bec570e4b02a
+    type: title
+    task:
+      id: b6d11f6e-a28a-459a-8004-bec570e4b02a
+      version: -1
+      name: Analysis
+      type: title
```
Comment (altmannyarden) [inline Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml:549]: Move this title to be right after the playbook trigger. 
+ all the tasks after the analysis should and can be parallel

File: Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml [hunk 15]
```diff
@@ -0,0 +1,1344 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "2":
+    id: "2"
+    taskid: 3ddedabb-1395-4234-81f5-a3b098a10721
+    type: condition
+    task:
+      id: 3ddedabb-1395-4234-81f5-a3b098a10721
+      version: -1
+      name: Is the causality process signed and prevalent?
```
Comment (altmannyarden) [inline Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml:101]: in the condition you are checking the opposite - if the process is not signed and not prevalent

File: Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml [hunk 16]
```diff
@@ -0,0 +1,1344 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "75":
+    id: "75"
+    taskid: a2bad801-d912-45ec-8ac6-5ece49400caa
+    type: condition
+    task:
+      id: a2bad801-d912-45ec-8ac6-5ece49400caa
+      version: -1
+      name: Is the XSIAM agent installed on the remote endpoint?
```
Comment (altmannyarden) [inline Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml:1052]: lets talk about it

File: Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml [hunk 17]
```diff
@@ -0,0 +1,1344 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "76":
+    id: "76"
+    taskid: 36164015-8f07-4e5b-873d-024f66adb228
+    type: condition
+    task:
+      id: 36164015-8f07-4e5b-873d-024f66adb228
+      version: -1
+      name: Is the XSIAM agent install and connected on the remote endpoint?
```
Comment (altmannyarden) [inline Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml:1093]: Is agent install and connected on the remote endpoint? or XDR agent. 
its not XSIAM agent :)

Comment (altmannyarden) [reply Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml:1093]: true for more tasks

File: Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml [hunk 18]
```diff
@@ -0,0 +1,1344 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "14":
+    id: "14"
+    taskid: 9de4fea1-2efe-427d-83e7-5ca0c0ffaff1
+    type: regular
+    task:
+      id: 9de4fea1-2efe-427d-83e7-5ca0c0ffaff1
+      version: -1
+      name: Disable the malicious scheduled task
```
Comment (altmannyarden) [inline Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml:337]: on remote host

File: Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml (hunk 17 above)
Comment (umishkin) [reply Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml:1093]: prefered first option, let's try to not mention XDR
```suggestion
      name: is agent install and connected on the remote endpoint?
```

File: Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task_README.md (hunk 13 above)
Comment (OmriItzhak) [reply Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task_README.md]: Done

File: Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task_README.md (hunk 12 above)
Comment (OmriItzhak) [reply Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task_README.md]: Done.

File: Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml (hunk 14 above)
Comment (OmriItzhak) [reply Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml]: Done.

File: Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml (hunk 15 above)
Comment (OmriItzhak) [reply Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml]: Done.

File: Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml (hunk 17 above)
Comment (OmriItzhak) [reply Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml]: Done.

File: Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml (hunk 18 above)
Comment (OmriItzhak) [reply Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml]: Done.

File: Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml [hunk 19]
```diff
@@ -0,0 +1,1344 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "5":
+    id: "5"
+    taskid: d6cce08c-349e-44db-807d-b6348886db73
+    type: title
+    task:
+      id: d6cce08c-349e-44db-807d-b6348886db73
+      version: -1
+      name: No Results Found
```
[the remark this answers is not in the document]
Comment (OmriItzhak) [reply Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml]: Done.

File: Packs/Core/ReleaseNotes/3_0_97.md [hunk 20]
```diff
@@ -0,0 +1,33 @@
...
+
+##### New: A source host created uncommon remote scheduled task
+
+New: This playbook handles "Uncommon remote scheduled task creation" alerts, which are generated on the source host that created the remote scheduled task.
+
+Playbook Stages:
+
+Analysis:
+
+- The playbook verifies whether the causality process is signed. If the process is not signed, it proceeds with remediation actions; otherwise, it continues investigating the alert.
```
Comment (umishkin) [inline Packs/Core/ReleaseNotes/3_0_97.md:12]: and pvevalent

File: Packs/Core/Triggers/Trigger_-_A_source_host_created_uncommon_remote_scheduled_task.json [hunk 21]
```diff
@@ -0,0 +1,19 @@
+{
+  "trigger_id": "df44be7092df167c73c5fa4eecc8977b",
+  "playbook_id": "A source host created uncommon remote scheduled task",
```
Comment (umishkin) [inline Packs/Core/Triggers/Trigger_-_A_source_host_created_uncommon_remote_scheduled_task.json]: don't forget to change after name change

File: Packs/Core/doc_files/A_source_host_created_uncommon_remote_scheduled_task.png
Comment (umishkin) [inline Packs/Core/doc_files/A_source_host_created_uncommon_remote_scheduled_task.png:1]: change

**Review by umishkin — APPROVED** [review]

File: Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml [hunk 22]
```diff
@@ -0,0 +1,1344 @@
...
+  contentitemfields: {}
+name: A source host created uncommon remote scheduled task
+description: |-
+  This playbook handles "Uncommon remote scheduled task creation" alerts, which are generated on the source host that created the remote scheduled task.
+
+  Playbook Stages:
+
+  Analysis:
+
+  - The playbook verifies whether the causality process is signed. If the process is not signed, it proceeds with remediation actions; otherwise, it continues investigating the alert.
```
[the remark this answers is not in the document]
Comment (OmriItzhak) [reply Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml]: Done.

File: Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task_README.md [hunk 23]
```diff
@@ -0,0 +1,65 @@
...
+
+---
+There are no inputs for this playbook.
+
+## Playbook Outputs
+
+---
+There are no outputs for this playbook.
+
+## Playbook Image
```
[the remark this answers is not in the document]
Comment (OmriItzhak) [reply Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task_README.md]: Done.

File: Packs/Core/ReleaseNotes/3_0_97.md (hunk 20 above)
Comment (OmriItzhak) [reply Packs/Core/ReleaseNotes/3_0_97.md]: Done.

File: Packs/Core/Triggers/Trigger_-_A_source_host_created_uncommon_remote_scheduled_task.json (hunk 21 above)
Comment (OmriItzhak) [reply Packs/Core/Triggers/Trigger_-_A_source_host_created_uncommon_remote_scheduled_task.json]: Done.

File: Packs/Core/doc_files/A_source_host_created_uncommon_remote_scheduled_task.png
Comment (OmriItzhak) [reply Packs/Core/doc_files/A_source_host_created_uncommon_remote_scheduled_task.png:1]: Done.

**Pull request merged**